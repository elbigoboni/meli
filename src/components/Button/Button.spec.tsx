import { render } from "@testing-library/react";
import React from "react";
import { Button } from ".";

describe("<Button />", () => {
  describe("Render", () => {
    test("should render correctly", () => {
      const { container } = render(<Button>Generic button</Button>);
      expect(container).toMatchSnapshot();
    });
  });
});
