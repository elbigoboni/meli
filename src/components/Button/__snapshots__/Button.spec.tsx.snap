// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Render should render correctly 1`] = `
<div>
  <button
    class="sc-bdfBwQ fQfPmw"
  >
    Generic button
  </button>
</div>
`;
